docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -838,6 +838,85 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_getFeeRefundTotalsByRecipient
+
+The `flashbots_getFeeRefundTotalsByRecipient` JSON-RPC method returns the total amount of fee refunds that have been processed for a specific address.  Our refund process calculates these values weekly.
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "flashbots_getFeeRefundTotalsByRecipient",
+  "params": [
+    {
+      recipient,       // String, the address to query for fee refunds
+    }
+  ]
+}
+```
+
+The response contains three fields:
+
+```json
+{
+  "pending":"0x17812ea4fbbe314",
+  "received":"0x108d1b27b63a213",
+  "maxBlockNumber":"0x13ddb08"
+}
+```
+
+- `pending`: the total amount of fee refunds not yet received by the recipient
```

Comment:
"the total amount of fee refunds that have been earned but not yet received by the recipient"